3/3.11/download_source

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -eu
```

Comment:
Good question. It's actually a bit tricky to check, since:
* some version folders contain additional files that are not in the `templates` one
* `3.10` is no longer in `versions.yaml` file so the verification code doesn't even look in there.

Let me know if you have ideas how to verify them.